cmd/kube-controller-manager/app/core.go

```diff
@@ -865,3 +865,43 @@ func startStorageVersionGarbageCollectorController(ctx context.Context, controll
 	).Run(ctx)
 	return nil, true, nil
 }
+
+func newSELinuxWarningControllerDescriptor() *ControllerDescriptor {
+	return &ControllerDescriptor{
+		name:                names.SELinuxWarningController,
+		aliases:             []string{"selinux-warning"},
+		initFunc:            startSELinuxWarningController,
+		isDisabledByDefault: true,
```

Comment:
The KEP says:
> This new controller will be opt-in, so it won't run on clusters that don't care about SELinux.

We can change that, still, I think it would be waste of CPU and memory for most Kubernetes distros.